service/src/test/java/bio/terra/cbas/runsets/outputs/StockOutputDefinitions.java

```diff
@@ -45,4 +45,21 @@ public static WorkflowOutputDefinition optionalOutputDefinition(
 
     return objectMapper.readValue(rawOutputDefinition, new TypeReference<>() {});
   }
+
+  public static WorkflowOutputDefinition arrayOutputDefinition(
```

Comment:
Would this belong in https://broadworkbench.atlassian.net/browse/WM-1512?

Reply:
Yeah, I ended up bundling them both into this PR since there was no additional coding needed (just test writing)